src/domain/registration/form/RegistrationForm.tsx

```diff
@@ -61,18 +60,28 @@ const RegistrationForm: FunctionComponent<Props> = ({
               setFormValues(values);
 
               // FIXME: Ensure that relationship is submitted to backend
-              const backendSupportChildren = values.children.map(child =>
-                omit(child, ['postalCode', 'homeCity', 'relationship'])
-              );
+              const backendSupportChildren = values.children.map(child => {
+                const c = omit(child, [
+                  'postalCode',
+                  'homeCity',
+                  'relationship',
+                ]);
+                // Here is the meat of relationshiop:
```

Comment:
bad comments

Reply:
We should actually change the relationships into array instead of doing duct-tape